spec/integ/matrix-client-crypto.spec.js

```diff
@@ -689,4 +689,62 @@ describe("MatrixClient crypto", function() {
                 return aliTestClient.httpBackend.flush('/keys/query', 1);
             });
     });
+
+    it("React on device_one_time_keys send by /sync", function() {
+        // Send a response which causes a key upload
+        const self = aliTestClient;
```

Comment:
using `self` instead of `aliTestClient` is unclear. I suggest you just use `aliTestClient` throughout.